<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="3" hostname="Lokeshs-Air" name="udemy.practice.JiraBugCreateTest" tests="4" failures="0" timestamp="21 Jun 2019 05:37:51 GMT" time="0.001" errors="1">
  <testcase name="deleteBug" time="0.000" classname="udemy.practice.JiraBugCreateTest">
    <skipped/>
  </testcase> <!-- deleteBug -->
  <testcase name="jiraAuthentication" time="0.001" classname="udemy.practice.JiraBugCreateTest">
    <error type="java.io.FileNotFoundException" message="/Users/lokesh/eclipse-workspace/RestAssured/src/test/java/config/env.properties (No such file or directory)">
      <![CDATA[java.io.FileNotFoundException: /Users/lokesh/eclipse-workspace/RestAssured/src/test/java/config/env.properties (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at udemy.practice.JiraBugCreateTest.jiraAuthentication(JiraBugCreateTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:626)
	at org.testng.TestRunner.run(TestRunner.java:594)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- jiraAuthentication -->
  <testcase name="addCommentToBug" time="0.000" classname="udemy.practice.JiraBugCreateTest">
    <skipped/>
  </testcase> <!-- addCommentToBug -->
  <testcase name="createJiraBug" time="0.000" classname="udemy.practice.JiraBugCreateTest">
    <skipped/>
  </testcase> <!-- createJiraBug -->
</testsuite> <!-- udemy.practice.JiraBugCreateTest -->
